app/models/attachment.rb

```diff
@@ -48,6 +48,14 @@ def self.parliamentary_sessions
     end
   end
 
+  def attachable_is_access_limited?
+    attachable && attachable.access_limited?
```

Comment:
Minor: Again it feels as if there's some duplication here. Could the implementation not be `access_limited_object.present?` in all scenarios.

Reply:
I'll try to remember to take a look at this when I revisit the code.